ui/docs/reference/integrations/github/checks.mdx

```diff
@@ -0,0 +1,44 @@
+# Taskcluster-Github Checks
+
+Taskcluster-Github supports [Github's Checks API](https://developer.github.com/v3/checks/).
+At the moment, the feature is in beta. If you would like to try it out:
+* Make sure you use `.taskcluster.yml` v1
+* Add `reporting: checks-v1` line at the root level of your `.taskcluster.yml`
+* You will need `queue:route:checks` scope for your `repo:github.com/org/repo:<action>` role.
+* If you are using a decision task, make sure you add `route.checks` to the `routes` of your tasks
+
+## Custom Output in Checks
+
+If you need to add some sort of custom output to your checks (for example, you would like to process
+your logs and add some results to the check's body):
+* You need to arrange for an artifact with the custom text (plain or Markdown) you'd like to add to your check 
+ (for the above example, you arrange the processing of your logs in the way you want, 
+ format the result the way you want using Markdown and save the result in an artifact)
```

Comment:
which example above?

Reply:
lines 12-13, `(for example, you would like to process your logs and add some results to the check's body)`

Reply:
I removed the "above example" tho - I agree it might be confusing. Thanks!